index.js

```diff
@@ -1 +1,22 @@
-// Code your solution in this file!
+const returnFirstTwoDrivers = function(array){
+  return array.slice(0,2);
+}
+const returnLastTwoDrivers = function(array){
+  return array.slice(2,4);
+}
+const selectingDrivers = [returnFirstTwoDrivers,returnLastTwoDrivers];
+
+function createFareMultiplier(multiplier){
+  return function (fare){
+    return fare * multiplier;
+  }
+}
+const fareDoubler = function(fare){
+  return (createFareMultiplier(2))(fare);
+}
```

Comment:
This can be refactored a bit:
```suggestion
function createFareMultiplier(multiplier){
  return fare * multiplier;
}
const fareDoubler = function(fare){
  return createFareMultiplier(fare);
}
```